src/main/java/org/gridsuite/sensitivityanalysis/server/computation/service/AbstractResultContext.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
```

Comment:
2024 for the new file

Reply:
Actually that file was created in 2023 (for the loadflow). This is just a copy that has to remain identical to the original.